master/buildbot/test/unit/test_data_resultspec.py

```diff
@@ -309,6 +309,11 @@ def test_popField_not_present(self):
 
 class NoneCmp(unittest.TestCase):
 
+    try:
+        cmpFunc = cmp
+    except NameError:
+        skip = "cmp() builtin not available on Python 3"
+
     def test_nonecmp(self):
```

Comment:
I'd believe nonecmp is not used anymore as we are switching the implementation to use key=, so you can remove the test and implementation